Q1/ash-13/Solution.py

```diff
@@ -0,0 +1,63 @@
+# Python3 implementation of the approach  
+  
```

Comment:
Please add `run.md`
Go through the README of this repo for more instructions regarding PR submission. @ash-13

Reply:
Sure bhai